dcompiler.dd

```diff
@@ -85,50 +85,109 @@ C:\&gt;unzip dmc.zip
 
     $(INSTALLATION_SCRIPT_HINT)
 
-    It will create
-    a $(D ~/dmd) directory with all the files in it.
+    Unzip the archive into your home directory.
+    It will create a $(D ~/dmd) directory with all the files in it.
 
     All the tools are command line tools, which means
-    they are run from a console window.)
-    $(LI All tools can be run directly from the archive:
+    they are run from a console window.
+    All tools can be run directly from the archive:
 $(CONSOLE
 ~/dmd/linux/bin64/dmd
 )
-    To install $(D dmd) globally, add this bin folder to your $(B PATH).
+    To install $(D dmd) globally, add the `~/dmd/linux/bin64` folder to your $(B PATH).
+$(CONSOLE
+export PATH="$HOME/dmd/linux/bin64:$PATH"
+)
+
+    Alternatively, you can install dmd locally yourself by:
+    $(P $(UL
+
+    $(LI Create a configuration file in $(D /etc/dmd.conf):
+
+---
+[Environment64]
+DFLAGS=-I/usr/local/include/d/dmd -L-L/usr/local/lib -L--export-dynamic -fPIC
+---
+
+    )
+
+    $(LI Copy binaries to $(D /usr/local/bin):
+
+$(CONSOLE
+sudo cp $(DMDDIR)/linux/bin64/{dmd,$(DUMPOBJ),$(OBJ2ASM),rdmd,ddemangle,dub,dustmite} /usr/local/bin
```

Comment:
I guess this is the only controversial part as it uses `bin64`, but as the existing example above (L95) already just shows 64-bit, I went for simplicity with 64-bit manual installation instructions only.
32-bit users are rare these days (some distros like mine don't support 32-bit anymore) and if they want to install dmd 32-bit globally, they know how to do it anyhow.
This is for the casual user, who for some reason, doesn't want neither the local folder nor the distro package globally installed.